tutorials/ph-mult-rasters/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Using Earth Engine to export multiple rasters based on user-defined dates
+description: In this tutorial, we will learn how to export multiple land surface temperature rasters for the same month over multiple years.
+author: hiyer09
+tags: exporting data, rasters, loops, Africa, public health
+date_published: 2019-09-18
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/2d3ad0c07630e858f0274d86a207d994)
+
+It is desirable to pull multiple raster images for environmental public health research. Typically the research question will guide the temporal resolution, but in most cases having multiple images over the course of the year is useful to interrogate possible seasonal patterns in exposure-disease relationships.
+
+In this tutorial, we will demonstrate one approach to pre-specifying the season, and then pulling one raster per season per year. For our exposure data, we will use land surface temperature data. We will conduct this analysis for Botswana.
+
+## Step 1: Draw a polygon around Botswana
+
+We can just use the polygon tool in the map window to draw a polygon. This polygon will be called "geometry" and be imported to the top of the program as an object. We can then define this object in our code as "region."
+```
+var region = geometry;
+
+```
+## Step 2: Set up date variables
+
+Let's say we are interested in pulling temperature data for the years 2000, 2001, and 2002. We can store this information in Google Earth Engine as an array:
+
+```
+// specify years -> can extend this code by adding years to the array
+
+var yrarr = ["2000", "2001", "2002"];
+print(yrarr);
+
+```
+Next, we need to define our start and end dates to filter the image collection. We can do this by defining functions. Note that you can adapt the array to store additional start and end dates for additional months (e.g. April, July, September) by adding additional elements to the arrays. We also want to iterate over years for this, so we are writing a function to add the year that we have predefined to the MM-DD strings.
+
+```
+
+// create array of dates for start -> can extend this code by adding "yr + "" to each element
+// in the array
+
+var makeSt = function(yr){
+  var stArr = [yr + "-01-01"]; 
+  
+  // print(stArr) - this is a sanity check to make sure it's doing what we think;
+  return(stArr);
+};
+
+// create array of dates for end
+var makeEd = function(yr){
+   var eArr = [yr + "-12-31"];
+   
+  //  print(eArr) - this is a sanity check to make sure it's doing what we think;
+   return(eArr);
+};
+
+```
+## Step 3: Write function to apply filters and processing data for export
+
+In this step, we are going to perform the following tasks in sequence as part of a function called "GetImage":
```

Comment:
Be sure to update `GetImage` to `getImage` throughout.

Reply:
Done